components/pages/Profile.js

```diff
@@ -34,7 +35,7 @@ export default function Profile() {
       <View style={{ flexDirection: 'row', justifyContent: 'space-between' }}>
         <Button title='Activity' onPress={() => setProfileSubPage('Activity')} />
         <Button title="Friends" onPress={() => setProfileSubPage('Friends')} />
-        <Button title="Communities" onPress={() => setProfileSubPage('Communities')} />
+        <Button title="Community" onPress={() => setProfileSubPage('ProfileCommunity')} />
```

Comment:
nitpick: I feel like this should be Communities, plural, since it is a list of communities that the user belongs to.

Reply:
on the user facing side I mean ^